about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <!-- CSS Stylesheet-->
+  <link rel="stylesheet" type="text/css" href="styles.css">
+
+  <!--Google Fonts-->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Fredoka+One&family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap"
+    rel="stylesheet">
+
+  <!-- Font Awesome icons -->
+  <script src="https://kit.fontawesome.com/11bb635b00.js" crossorigin="anonymous"></script>
+
+  <title>About</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="openMenu hamburger-container icon"><a class="hamburger">≡</a></div>
+      <a href="index.html"><img class="logo" src="images/Orange.png" alt="Orange Studio Logo"></a>
+      <ul class="mainMenu navbar-links" id="mainmenu">
+        <li><a href="about.html" class="navlink">About Us</a></li>
+        <li><a href="#services" class="navlink">Our Services</a></li>
+        <li><a href="#contact" class="navlink">Contact Us</a></li>
+        <li>
+          <div class="closeMenu"><i class="fas fa-times icons"></i></div>
+        </li>
+      </ul>
+    </nav>
+  </header>
```

Comment:
- [ ] The navbar on both the home page and about page is divided into a top dark bar and a bottom white bar where the top short bar section is made up of social media links, logout, and language links aligned to the right and the taller bottom bar with a logo and a list of page links. Please kindly fix that.
- [ ] On the bottom white bar the logo is on the left side and the list of links is on the right side. Please kindly fix that.